modules/gateway.go

```diff
@@ -22,76 +22,38 @@ var (
 	BootstrapPeers = build.Select(build.Var{
 		Standard: []NetAddress{
 			"101.200.214.115:9981",
-			"104.223.98.174:9981",
 			"109.172.42.157:9981",
-			"109.206.33.225:9981",
-			"109.71.42.163:9981",
-			"109.71.42.164:9981",
 			"113.98.98.164:9981",
-			"115.187.229.102:9981",
-			"120.25.198.251:9981",
-			"138.201.13.159:9981",
 			"139.162.152.204:9981",
-			"141.105.11.33:9981",
 			"142.4.209.72:9981",
 			"148.251.221.163:9981",
-			"158.69.120.71:9981",
 			"162.210.249.170:9981",
 			"162.222.23.93:9981",
-			"176.9.59.110:9981",
 			"176.9.72.2:9981",
-			"180.167.17.236:9981",
 			"18.239.0.53:9981",
-			"183.86.218.232:9981",
 			"188.166.61.155:9981",
 			"188.166.61.157:9981",
 			"188.166.61.158:9981",
-			"188.166.61.159:9981",
-			"188.166.61.163:9981",
-			"188.61.177.92:9981",
 			"190.10.8.173:9981",
-			"193.198.102.34:9981",
 			"194.135.90.38:9981",
 			"195.154.243.233:9981",
-			"202.63.55.79:9981",
 			"210.14.155.90:9981",
 			"213.251.158.199:9981",
 			"217.65.8.75:9981",
-			"222.187.224.89:9981",
-			"222.187.224.93:9981",
 			"23.239.14.98:9971",
 			"23.239.14.98:9981",
-			"23.239.14.98:9981",
-			"24.91.0.62:9981",
-			"31.178.227.21:9981",
-			"37.139.1.78:9981",
 			"37.139.28.207:9981",
-			"43.227.113.131:9981",
-			"45.79.132.35:9981",
 			"45.79.159.167:9981",
 			"46.105.118.15:9981",
-			"52.205.238.6:9981",
-			"62.107.201.170:9981",
-			"62.210.207.79:9981",
-			"62.212.77.12:9981",
 			"64.31.31.106:9981",
-			"68.55.10.144:9981",
 			"73.73.50.191:33721",
-			"76.164.234.13:9981",
 			"78.119.218.13:9981",
 			"79.172.204.10:9981",
-			"79.51.183.132:9981",
 			"80.234.37.94:9981",
 			"82.196.11.170:9981",
 			"82.196.5.50:9981",
 			"82.220.99.82:9981",
-			"83.76.19.197:10981",
 			"85.255.197.69:9981",
-			"87.236.27.155:12487",
-			"87.98.216.46:9981",
-			"88.109.8.173:9981",
-			"91.121.183.171:9981",
-			"95.211.203.138:9981",
```

Comment:
worth pointing out that every node that remains has been there for 4 months or more, as that's when the list was last added to. But I only added nodes which had already been around for a while.

Which means we have a decent list of long term nodes :)